### ISSUE TITLE
performance testing result is not as expected

### ISSUE THREAD

**Issue by imliuda** [timescale/timescaledb#494]
hi, i have tested timescaledb on aliyun cloud ecs.g5.large server. the server configuration is as follows:

* 2 cpu (Intel Xeon(Skylake) Platinum 8163)
* 8GB memory
* 2 100GB network ssd disks (1800 IOPS)
* Debian 9.2 OS
* postgresql version 9.6

and my postgresql.conf is as follows:

```ini
shared_preload_libraries = 'timescaledb'
data_directory = ‘/data/pgdb’
shared_buffers = 2GB
```

i mounted 2 disks at /data and /data1, tesing data is in /data1.

I got this result:

![image](https://user-images.githubusercontent.com/11783644/38225194-4e4e1692-3726-11e8-8be7-9b83168d9339.png)

the speed of timescaledb is unstable,  there is great fluctuation. i have run this command:

```
benchmark=# select * from chunk_relation_size('cpu_ts');
```

i found there are 6 chunks. my quanstion is if the result is normal? how can i get a stable insert speed?

**Comment by mfreed** [comment]
Hi @imliuda :

Can you describe your benchmarking setup a bit more?  I.e., what does your table schema look like (`\d+`), how do you insert data, what timestamps are chosen for inserted data?

Further, can you print the full output of `select * from chunk_relation_size_pretty('cpu_ts');`?

My immediate guess is that your chunk size is too large (a configuration option to `create_hypertable`), so want to dig into this.

As to your question, this result is not normal with a proper setup.  See:  https://blog.timescale.com/timescaledb-vs-6a696248104e

**Comment by imliuda** [comment]
sorry, I made this test according to [this document](https://github.com/timescale/benchmark-postgres/blob/master/README.md). after I read that article, I retested. I found the result is same. this is my steps: [timescale-test-command.txt](https://github.com/timescale/timescaledb/files/1871517/timescale-test-command.txt).

I make the chart using per-minute report statics, may be different from yours, so the line is not smooth. the most import thing is that the data i use is not large enough, it only has 100 million records. in your test, you have 1 billion records. in your ingest chart, when number of record is 100 million, the speed of timescaledb is only two times of postgresql, almost the same with my result. so when data is large enough, i think i will get the same result with you. 

in my result, according to the periodic chage of ingest rate, i can clear see that a new chunk must be created.

here, i want to remind others who want to make a test, if you data isn't large enough, i advise you use a 
server with poor performance. e.g, one core cpu and 1GB memory and a sata disk. you will get the expected result.

thanks @mfreed very much.
